Files/ViewModels/MainPageViewModel.cs

```diff
@@ -159,6 +163,21 @@ private async void AddNewInstanceAccelerator(KeyboardAcceleratorInvokedEventArgs
             e.Handled = true;
         }
 
+        private void ToggleFullScreenAccelerator(KeyboardAcceleratorInvokedEventArgs e)
+        {
+            ApplicationView view = ApplicationView.GetForCurrentView();
+            if (view.IsFullScreenMode)
+            {
+                view.ExitFullScreenMode();
+            }
+            else
+            {
+                view.TryEnterFullScreenMode();
+            }
+
+            e.Handled = true;
+        }
```

Comment:
Should this be in `MainPage.xaml.cs`?

Reply:
Oops yeah. Will move to `MainPage.xaml.cs`

Reply:
@yaichenbaum Should I move [`ICommand` property](https://github.com/files-community/Files/blob/a98b5f28707332ba54ecdbcdc0217a20d85609e0/Files/ViewModels/MainPageViewModel.cs#L51) to `MainPage.xaml.cs` as well?

Reply:
For now, yes. Eventually commands like this can be grouped into their own ViewModel so that we can access them from any page.